examples/04_pong/main.rs

```diff
@@ -335,6 +338,20 @@ impl State for Pong {
                 bounce_sfx,
                 score_sfx,
             });
+            // Sets the cursor hidden when the mouse is hovering the pong window.
+            if let Some(mut window_events) = engine
+                .world
+                .res
+                .try_fetch_mut::<EventHandler<WindowModifierEvent>>(0)
+            {
+                window_events.write_single(WindowModifierEvent {
+                    modify: |win| {
+                        win.set_cursor_state(winit::CursorState::Hide)
+                            .ok()
+                            .expect("could not grab mouse cursor")
```

Comment:
How would this be handled in a real world application? Shouldn't we use best practices in our examples?

Reply:
I'm not sure how that could actually crash. What I'm sure though, is that the error message is not the right one.